apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -0,0 +1,407 @@
+        /// The maximum number of cells allowed per row.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter CellsPerRowLimit(int limit)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which skips the first N cells
+        /// of each row, matching all subsequent cells.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// If duplicate cells are present, as is possible when using an Interleave,
+        /// each copy of the cell is counted separately.
+        /// </para>
+        /// </remarks>
+        /// <param name="limit">
+        /// The maximum number of cells allowed per row.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter CellsPerRowOffset(int limit)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which sends rows through
+        /// several RowFilters in sequence.
+        /// </summary>
+        /// <param name="filters">
+        /// The elements of "filters" are chained together to process the input row:
+        /// in row -> f(0) -> intermediate row -> f(1) -> ... -> f(N) -> out row
+        /// The full chain is executed atomically.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter Chain(params RowFilter[] filters)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells from
+        /// columns whose qualifiers satisfy the given RE2 regex.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that, since column qualifiers can contain arbitrary bytes, the `\C`
+        /// escape sequence must be used if a true wildcard is desired. The `.`
+        /// character will not match the new line character `\n`, which may be
+        /// present in a binary qualifier.
+        /// </para>
+        /// </remarks>
+        /// <param name="regex">
+        /// The RE2 regex used to match column qualifiers.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter ColumnQualifierRegex(ColumnQualifier regex)
+        {
+            throw new InvalidOperationException();
+        }
+
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells from
+        /// columns within the given range.
+        /// </summary>
+        /// <param name="range">
+        /// The range of columns from which cells should be matched.
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter ColumnRange(ColumnRange range)
```

Comment:
Remove this, given the OpenClosed methods etc?

Reply:
I'm assuming this no longer applies, but correct me if I'm wrong.